iree/compiler/Conversion/LinalgToLinalg/Conv2DToImg2Col.cpp

```diff
@@ -0,0 +1,191 @@
+// Copyright 2020 Google LLC
```

Comment:
Minor comment. Is this adapted from this https://github.com/google/iree/blob/main/iree/compiler/Conversion/LinalgToLLVM/ConvImg2ColMatmulConversion.cpp . 

We could remove that file cause it was not really used on the buffer path anyway

Reply:
Yes this is adapted, I want to remove that pass later on but its ok just did here :)